src/Akeneo/Connectivity/Connection/back/Infrastructure/Apps/Persistence/DenormalizeConnectedAppTrait.php

```diff
@@ -40,6 +41,7 @@ private function denormalizeRow(array $dataRow): ConnectedApp
             (bool) $dataRow['certified'],
             $dataRow['partner'],
             (bool) ($dataRow['is_test_app'] ?? false),
+            (bool) $dataRow['has_outdated_scopes'],
```

Comment:
```suggestion
            (bool) ($dataRow['has_outdated_scopes'] ?? false) ,
```
Trait used by multiple queries that might not specify